tests/test_codesigned.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+from __future__ import annotations
+
+import logging
+import os
+from functools import lru_cache
+from pathlib import Path
+from shutil import which
+from subprocess import CalledProcessError, check_call, check_output
+
+import pytest
+
+from conda.common.compat import on_win
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+REPO_ROOT = (Path(HERE) / "..").resolve().absolute()
+STUB_FOLDER = REPO_ROOT / "conda" / "shell"
+
+
+@lru_cache(maxsize=None)
+def find_signtool() -> str | None:
+    """Tries to find signtool
```

Comment:
Is there any documentation to this signtool program you could link to here? I'm thinking this might be nice for future developers who need to debug this when it starts failing.